src/scripts/services/dimStoreService.factory.js

```diff
@@ -642,6 +645,29 @@ const _ = require('underscore');
         return null;
       }
 
+      if (itemDef.classified) {
+        var language = dimSettingsService.language;
+        if (dimClassifiedData[itemDef.itemHash]) { // do we have declassification info for item?
+           // itemDef.icon = dimClassifiedData[itemDef.itemHash].icon;
+          itemDef.itemName = dimClassifiedData[itemDef.itemHash].i18n[language].itemName;
```

Comment:
If we remove i18n and just provide classified items in English `angular.extend` will work amazingly... In order to provide the i18n translations I have been manually changing the language of my ps4 if we can provide someway for submitting classified items that the dev team does not possess and also provide the translations that would be even more excellent... But yes I am committed to maintaining this list, as can be seen by the upkeep of #1262...